UI/MVUX/MVUX/MVUX/Presentation/StateSample/StatePage.xaml

```diff
@@ -0,0 +1,26 @@
+﻿<Page x:Class="MVUX.Presentation.StateSample.StatePage"
+	  xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	  xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	  xmlns:local="using:MVUX.Presentation.StateSample"
+	  xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	  xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	  xmlns:utu="using:Uno.Toolkit.UI"
+	  mc:Ignorable="d"
+	  d:DesignHeight="300"
+	  d:DesignWidth="400">
+
+  <StackPanel utu:SafeArea.Insets="Bottom,Top">
+
+		<TextBlock Text="This sample displays uses an IState &lt;T> where T is a record type holding the number of attendants present in the hall, to get or set the crowdedness in a wedding hall."
+				   Margin="10"
+				   FontSize="16"
+				   TextWrapping="Wrap" />
+	  
+		<TextBlock Text="How many people are currently in the hall?" />
+	<TextBox DataContext="{Binding HallCrowdedness}"
+			 Text="{Binding NumberOfPeopleInHall, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" />
+
+	<Button Content="Save"
+			Command="{Binding Save}" />
+  </StackPanel>
```

Comment:
I feel like we don't need this sample as we are showing how to update and display value of a state in the How to Update a State sample section